src/firefly/java/edu/caltech/ipac/firefly/server/events/CacheEventWorker.java

```diff
@@ -20,18 +20,17 @@
 import net.sf.ehcache.event.CacheEventListener;
 import net.sf.ehcache.event.NotificationScope;
 
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-
 import static edu.caltech.ipac.firefly.server.ServerContext.SHORT_TASK_EXEC;
 
 /**
  * @author Trey Roby
+ *
+ * @deprecated - no longer using replicating cache;  use {@link edu.caltech.ipac.firefly.server.events.MessageEventWorker} instead
  */
+@Deprecated
 public class CacheEventWorker implements ServerEventManager.EventWorker {
```

Comment:
should this file be deleted?

Reply:
We’ve just deprecated it in this release. Let’s remove it later in future updates.